CMakeLists.txt

```diff
@@ -141,4 +141,4 @@ target_include_directories(astronomy_include_directories
 enable_testing()
 if(BOOST_ASTRONOMY_BUILD_TEST)
 	add_subdirectory(test)
-endif()
+endif()
```

Comment:
nitpicking: the empty end line is missing

Reply:
Resolved.